extensions/wikia/AdEngine/js/provider/directGpt.js

```diff
@@ -61,8 +61,11 @@ define('ext.wikia.adEngine.provider.directGpt', [
 			isPageFairRecoverable: pageFair ? pageFair.isSlotRecoverable : false,
 			sraEnabled: sraEnabled,
 			atfSlots: atfSlots,
-			getAdUnitBuilder: function () {
-				return context.opts.megaAdUnitBuilderEnabled ? megaAdUnitBuilder : kiloAdUnitBuilder;
+			megaAdUnitSlots: ['INVISIBLE_SKIN'],
+			getAdUnitBuilder: function (slotName) {
```

Comment:
Can we unittest it somehow?
Also please remember we serve `INVISIBLE_SKIN` in Remnant provider

Reply:
Done.